### ISSUE TITLE
Inflow or Fluid UI options are not fully displayed if a FLIP Domain is not in the scene

### ISSUE THREAD

**Issue by rlguy** [rlguy/Blender-FLIP-Fluids#629]
#### System and Blend File Information

**Blender Version:** 3.6.5 Release Candidate, blender-v3.6-release, 2023-10-12 12:38, 521a68323c53
**Addon Version:** A FLIP Fluid Simulation Tool for Blender (v1.7.2 Release 07-NOV-2023)
**OS:** Windows-10-10.0.22621-SP0
**GPU:** NVIDIA GeForce RTX 4090/PCIe/SSE2 NVIDIA Corporation 4.5.0 NVIDIA 536.99
**CPU:** 13th Gen Intel(R) Core(TM) i9-13900K
**CPU Threads:** 32
**RAM:** 64GB

#### Describe the bug

In FLIP Fluids 1.7.2, If a FLIP Domain has not yet been created the Inflow and Fluid UI generates an error that prevents other options from being drawn. This issue is only present when the _FLIP Fluids Preferences > Enable Developer Tools_ option is enabled.

Blender System Console error message:

```
Traceback (most recent call last):
  File "C:\Users\ryanl\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\flip_fluids_addon\ui\inflow_ui.py", line 145, in draw
    is_color_attribute_enabled = (dprops.surface.enable_color_attribute or
AttributeError: 'NoneType' object has no attribute 'surface'
```

#### To Reproduce

1. Create new scene
2. Create a FLIP Inflow object
3. View Inflow UI

[inflow_bug_example.zip](https://github.com/rlguy/Blender-FLIP-Fluids/files/13321898/inflow_bug_example.zip)

#### Expected Behaviour

Expected Inflow UI display:

![image](https://github.com/rlguy/Blender-FLIP-Fluids/assets/4285191/242a3d4b-c9ea-4f67-b5fc-4ce92bb84d5e)

#### Actual Behaviour

Actual Inflow UI display:

![image](https://github.com/rlguy/Blender-FLIP-Fluids/assets/4285191/611014e1-0db0-4d5b-a373-59ff3375ffc7)

#### Workaround

The current workaround to this issue is to create a FLIP Domain object. Afterwards, all settings will be displayed.

**Comment by rlguy** [comment]
This issue has been fixed and will be included in a FLIP Fluids 1.7.3 version update to be released within the next two weeks.

**Comment by rlguy** [comment]
This fix is now available in FLIP Fluids version 1.7.3!
